lms/djangoapps/verify_student/tests/test_services.py

```diff
@@ -187,3 +189,29 @@ def test_not_in_verified_track(self):
         service = ReverificationService()
         status = service.get_status(self.user.id, unicode(self.course_id), self.final_checkpoint_location)
         self.assertEqual(status, service.NON_VERIFIED_TRACK)
+
+    def test_get_course_verification_status(self):
+        # test that we can retrieve the course verification status via the service
+
+        service = ReverificationService()
+
+        # NEGATIVE TEST: we get back a None if we ask about a course that the user
+        # is not enrolled in
+        not_enrolled_course = CourseFactory.create(org='Robot', number='101', display_name='Second Test Course')
+        self.assertIsNone(
+            service.get_course_verification_status(
+                self.user.id,
+                not_enrolled_course.id
+            )
+        )
+
+        # NEGATIVE TEST: we get back a None if we ask about a course that the user
+        # is not enrolled as verified
+        self.enrollment.update_enrollment(mode=CourseMode.HONOR)
+        self.assertIsNone(
+            service.get_course_verification_status(self.user.id, self.course_id)
+        )
+
+        self.enrollment.update_enrollment(mode=CourseMode.VERIFIED)
```

Comment:
Maybe split into 2 methods, 1 for positive and 1 for negative. At the very least, add comment of `# POSITIVE TEST` to make it more quickly clear you are covering both.

:+1: after minor change. Thank you.